packages/property-grid/src/components/PropertyGrid.scss

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+@import "~@bentley/ui-core/lib/ui-core/style/index";
+@import "~@bentley/ui-core/lib/ui-core/text/index";
+
+.property-grid-property-panel-label {
+  @include uicore-text(leading);
+  font-size: $uicore-font-size-leading;
+  padding: $uicore-xs 0;
+}
+
+.property-grid-property-panel-class {
+  @include uicore-text;
+  font-size: $uicore-font-size;
+}
+
+.property-grid-navigator-property-grid {
```

Comment:
id remove navigator in the class name, and maybe clean up some of the others as well